hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -306,7 +311,8 @@ private void putChunksInBlock(int numOfChunksPerBlock, int i,
     long chunkLength = 100;
     try {
       for (int k = 0; k < numOfChunksPerBlock; k++) {
-        final String chunkName = String.format("block.%d.chunk.%d", i, k);
+        final String chunkName = String.format("%d_chunk_%d_block_%d",
```

Comment:
Why is the chunk name changed?

Reply:
If  FILE_PER_CHUNK, the chunk name rule is ```localID_chunk_chunkIndex```
If  FILE_PER_BLOCK,  the block name rule is ```localID + .block```
This convention is used by this patch, so here we need to adopt the convention to test the logic.

Reply:
Can you keep both schema in this test? i.e. Don't remove the old test.

Reply:
Both of the chunk name works fine with the parameterized test,  the old chunk name format is not working in the real cluster, no need to keep it.